pandas/core/config_init.py

```diff
@@ -239,10 +239,16 @@
 
 
 def mpl_style_cb(key):
+    import warnings
     import sys
     from pandas.tools.plotting import mpl_stylesheet
     global style_backup
 
+    msg = """
```

Comment:
you need to something like this in ``pandas/core/config_init.py``
rather than this directly.

```
cf.deprecate_option('display.line_width',
                    msg=pc_line_width_deprecation_warning,
                    rkey='display.width')
```